### PR TITLE
MAIN - restructures some parts of the project 

### PR DESCRIPTION
**Pull request by paschalidi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,25 +17,20 @@
   "scripts": {
     "linter:fix": "prettier --write '{,!(node_modules|.git|storybook-static)/**/}*.{js,json,jsx,md}'",
     "linter:check": "eslint stories/ && eslint src/",
-    "storybook": "start-storybook -p 3006 -s ./public",
-    "release": "node build.js",
-    "release:test": "node build.js test",
+    "serve": "start-storybook -p 3006 -s ./public",
```
Comment (herman-rogers) [inline package.json]: make sure the infrastructure we're using is also updated with this command (shouldn't be used but double checking can't hurt).

File: src/Input/input.style.js [hunk 2]
```diff
@@ -15,7 +9,6 @@ export const InputStyles = styled.div`
     font-stretch: normal;
     line-height: 2.5;
   }
-
```
Comment (herman-rogers) [inline src/Input/input.style.js]: leave the spacing hear - otherwise its harder to reason about.

File: src/MedicalCard/MedicalCard.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import SimpleIcon from "./SimpleIcon";
 import DetailedIcon from "./DetailedIcon";
 import missingAvatar from "../../public/images/empty-avatar.svg";
 import MedicalCardStyles from "./medicalCard.style";
-import DateDisplay from "../dates/DateDisplay";
+import { DateDisplay } from "../index";
```
Comment (herman-rogers) [inline src/MedicalCard/MedicalCard.jsx]: We shouldn't be doing this. It should be ideally ```import DateDisplay from "../dates/``` or the original form. I know the idea is to enforce using the index but we should do this inside the stories, not inside our actual source.

File: stories/Buttons.story.jsx [hunk 4]
```diff
@@ -3,9 +3,7 @@ import styled from "styled-components";
 import { storiesOf } from "@storybook/react";
 import { withInfo } from "@storybook/addon-info";
 import { action } from "@storybook/addon-actions";
-import Button from "../src/button/Button";
-import AnchorButton from "../src/button/AnchorButton";
-import Icon from "../src/icons";
+import { Button, AnchorButton, Icon } from "../src/index";
```
Comment (herman-rogers) [inline stories/Buttons.story.jsx]: this should be enough to guarantee we're exporting our components. We should do this in source, however.

Comment (paschalidi) [reply stories/Buttons.story.jsx]: in source?

File: src/MedicalCard/MedicalCard.jsx (hunk 3 above)
Comment (paschalidi) [reply src/MedicalCard/MedicalCard.jsx]: I will revert it, but it is the same thing. index.js is being bundled and served and it is already there locally.

File: stories/Buttons.story.jsx (hunk 4 above)
Comment (herman-rogers) [reply stories/Buttons.story.jsx]: shouldnt* do this in source, however

File: src/MedicalCard/MedicalCard.jsx (hunk 3 above)
Comment (herman-rogers) [reply src/MedicalCard/MedicalCard.jsx]: well in normal source we wouldn't use a concept like this, index.js only exists to serve the library / packaging tools we're using. Its not unfeasible that these tools would change and require, for example, a index.src.js file or something.

File: package.json (hunk 1 above)
Comment (paschalidi) [reply package.json]: ✔️

File: stories/Buttons.story.jsx (hunk 4 above)
Comment (paschalidi) [reply stories/Buttons.story.jsx]: ✔️

**Review by herman-rogers — APPROVED** [review]

**Review by herman-rogers — CHANGES_REQUESTED** [review]
Remove all API changes

**Review by herman-rogers — APPROVED** [review]

**Pull request merged**